### ISSUE TITLE
tests have become a bit noisy

### ISSUE THREAD

**Issue by mr-c** [dib-lab/khmer#907]
``` bash
(testenv1)mcrusoe@athyra:/tmp/tmp.QkpW7325RR/testenv1$ nosetests khmer --attr '!known_failing'
...........................................................................................................ERROR: Input file /tmp/khmertest_L78h8t/thisfiledoesnotexist does not exist
Exiting
.ERROR: Input file /tmp/khmertest_Hck90q/thisfiledoesnotexist does not exist
.........................................................................................................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 517 tests in 22.061s

OK
```

This happened somewhere around commit a7e0205b

**Comment by anotherthomas** [comment]
will be fixed by #922

**Comment by mr-c** [comment]
As @anotherthomas and I discussed this will not be fixed by #922, alas.

**Comment by ctb** [comment]
Fixed by #1137.
